ansible/roles/enomoloch-get-pcaps/files/get_tcp_dump_messages.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=getting enodump
+
+[Service]
+Restart=always
+TimeoutStartSec=300
+
+ExecStart=/services/sh get_messages.sh
```

Comment:
That doesn't look right

Reply:
Can you be a bit more specific?

Reply:
Should be something like  /pathtoscript/script.sh i think?

Reply:
Any update here?